services/http.go

```diff
@@ -152,6 +151,7 @@ func (s *httpService) Handle(ctx context.Context, conn net.Conn) error {
 			return err
 		}
 
+		sConn := s.scr.GetConnection("http", conn)
```

Comment:
Moving this into the for-loop for the http handler doesn't make sense, what would make this nessecary?

Reply:
This is necessary for reloading of the scripts, when a user does not leave the website, the connection is not closed. Changes made to scripts do not resolve to the end user when the connection is retrieved outside of the loop.

Reply:
We could change the method we reload the scripts. I think it would be better to change the luaState at the pointer location, instead of having the pointer change to a different address. This will eliminate the need of having to `GetConnection` every iteration of the for-loop.

Reply:
Why wouldn't changes to the scripts not be added to the scripts when outside of the for? They work on pointer base if I am correct and all pointers are stored in the scripter so can be called for a reload from outside.